src/pages/member/index.astro

```diff
@@ -23,7 +23,7 @@ const memberCollection = await getCollection('member')
                 <Image
                   src={member.data.image}
                   alt={member.data.name + " Avatar"}
-                  class="size-28 ring-base-content ring-offset-base-100 rounded-full ring-2 ring-offset-2 m-2"
+                  class="size-28 ring-base-content ring-offset-base-100 rounded-full ring-2 ring-offset-2 m-2" style="object-fit: cover;"
```

Comment:
added this because my image isn't square, and was getting stretched. You may want to move it to a css class somewhere, but I didnt know the best place